depends/packages/qrencode.mk

```diff
@@ -3,20 +3,22 @@ $(package)_version=4.1.1
 $(package)_download_path=https://fukuchi.org/works/qrencode/
 $(package)_file_name=$(package)-$($(package)_version).tar.bz2
 $(package)_sha256_hash=e455d9732f8041cf5b9c388e345a641fd15707860f928e94507b1961256a6923
+$(package)_patches=cmake_fixups.patch
 
 define $(package)_set_vars
-$(package)_config_opts=--disable-shared --without-tools --without-tests --without-png
-$(package)_config_opts += --disable-gprof --disable-gcov --disable-mudflap
-$(package)_config_opts += --disable-dependency-tracking --enable-option-checking
+$(package)_config_opts := -DWITH_TOOLS=NO -DWITH_TESTS=NO -DGPROF=OFF -DCOVERAGE=OFF
+$(package)_config_opts += -DCMAKE_DISABLE_FIND_PACKAGE_PNG=TRUE
```

Comment:
Nit: Could set `WITHOUT_PNG=ON` instead?

Reply:
I started there, but from what I can tell, that didn't seem to fully work? You'd also still end up with all the output at the end of configuring, i.e:
```bash
-- Found PNG: /opt/homebrew/lib/libpng.dylib (found version "1.6.43")
<snip>
--  .. PNG ......................... = TRUE
--  .... PNG includes .............. = /opt/homebrew/include;/Library/Developer/CommandLineTools/SDKs/MacOSX14.4.sdk/usr/include
--  .... PNG library ............... = /opt/homebrew/lib/libpng.dylib;/Library/Developer/CommandLineTools/SDKs/MacOSX14.4.sdk/usr/lib/libz.tbd
```
which could be confusing. Which is why I opted for `CMAKE_DISABLE_FIND_PACKAGE_PNG` instead.
```bash
--  .. PNG ......................... = 
--  .... PNG includes .............. = 
--  .... PNG library ............... = 
```

Reply:
If it's down to the output why not do both so we also get:
```
--  Project configuration:
--  .. Build test programs  ........ = OFF
--  .. Build utility tools  ........ = OFF
--  .. Disable PNG support  ........ = ON
```

Reply:
Fair enough. Have added that now.